multichain-api-gateway/src/config.rs

```diff
@@ -0,0 +1,38 @@
+use std::net::SocketAddr;
+
+use url::Url;
+
+/// An instance of the maintained networks in Blockscout.
+/// Semantic: (network, chain)
+/// e.g."blockscout.com/eth/mainnet" -> ("eth", "mainnet")
+#[derive(serde::Deserialize, Clone, Debug, PartialEq)]
+pub struct Instance(pub String, pub String);
+
+/// Settings for the Blockscout API
+#[derive(serde::Deserialize, Clone)]
+pub struct BlockScoutSettings {
+    /// The base URL of the Blockscout API.
+    /// At the moment, should be equal to "https://blockscout.com"
+    pub base_url: Url,
+
+    pub instances: Vec<Instance>,
+
+    /// The number of concurrent requests to be made to the Blockscout API from a server thread worker.
+    pub concurrent_requests: usize,
+}
+
+#[derive(serde::Deserialize, Clone)]
+pub struct ServerSettings {
+    pub addr: SocketAddr,
+}
+
+#[derive(serde::Deserialize, Clone)]
+pub struct Settings {
+    pub server: ServerSettings,
+    pub blockscout: BlockScoutSettings,
+}
```

Comment:
An interesting problem with that solution is that the service will fail to run if any of the fields would be missing and that is probably not the behavior you want (e.g. `blocscout.base_url` may be missing but we may safely run the service with `https://blockscou.com` as a default value). You may look at https://serde.rs/field-attrs.html#default for example, as one of the solutions